tsconfig.base.json

```diff
@@ -3,7 +3,7 @@
     "lib": [
       "es2020"
     ],
-    "target": "es6",
+    "target": "es2020",
```

Comment:
Changed target to `es2020` to properly support certain functionality in TypeScript.

The [testing](https://github.com/DeFiCh/jellyfish/blob/main/packages/jellyfish-testing/src/testing.ts#L14-L19) is using some functionality that errors using `es6`. Error is `Property '$1' is used before its initialization. ts(2729)`

Following the trail of the following typescript [ticket 1](https://github.com/microsoft/TypeScript/issues/37573) [ticket 2](https://github.com/microsoft/TypeScript/issues/44298) and documentation [useDefineForClassFields](https://www.typescriptlang.org/tsconfig#useDefineForClassFields).

When setting the base target to `es2020` the error goes away as expected.

Reply:
https://github.com/DeFiCh/jellyfish/pull/902

We had to set back to "es6" because downstream (React Native + Expo) couldn't compile. Would it be possible to set this for just this project?

Reply:
I think that's also the reason why we set target to "esnext" for `tsconfig.json` when developing.

Reply:
Noted, thanks for the reference. Let me do some more testing on this to confirm.

Reply:
@fuxingloh that worked fine, reverted tsconfig.base back to `es6` and set `es2020` for tsconfig which removes any development side complaints for testing. `esnext` kicks of with errors on that functionality.